cmd/migrate.go

```diff
@@ -0,0 +1,107 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"go/types"
+	"strconv"
+
+	migrate "github.com/rubenv/sql-migrate"
+	"github.com/spf13/cobra"
+	"github.com/stellar/go/support/config"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/wallet-backend/internal/db"
+)
+
+type migrateCmd struct{}
+
+func (c *migrateCmd) Command() *cobra.Command {
+	var databaseURL string
+	cfgOpts := config.ConfigOptions{
+		{
+			Name:        "database-url",
+			Usage:       "Database connection URL",
+			OptType:     types.String,
+			ConfigKey:   &databaseURL,
+			FlagDefault: "postgres://postgres@localhost:5432/wallet-backend?sslmode=disable",
+			Required:    true,
+		},
+	}
+
+	migrateCmd := &cobra.Command{
+		Use:   "migrate",
+		Short: "Schema migration helpers",
+		PersistentPreRun: func(_ *cobra.Command, _ []string) {
+			cfgOpts.Require()
+			if err := cfgOpts.SetValues(); err != nil {
+				log.Fatalf("Error setting values of config options: %s", err.Error())
+			}
+		},
```

Comment:
Before you write too many CLI commands, there's a good practice you should adopt here.

These commands have counterparts that return errors:
- `PersistentPreRun` -> `PersistentPreRunE`
- `Run` -> `RunE`
- ...

Instead of logging with Fatal, it's better to return an error and log + Panic in the root caller. It's easier to debug and to write tests for it.

`log.Fatalf` calls os.Exit(0), which terminates the program without very good feedback.

Reply:
Thanks for raising this @marcelosalloum! Great suggestion. 
I'll add to this one and refactor the other two commands we already have in place. 👍

Reply:
[2e5e3d0](https://github.com/stellar/wallet-backend/pull/18/commits/2e5e3d01baf15920f91f889987766224a92b723e)[2e5e3d0](https://github.com/stellar/wallet-backend/pull/18/commits/2e5e3d01baf15920f91f889987766224a92b723e)